queries.sql

```diff
@@ -14,4 +14,72 @@ SELECT * from animals WHERE neutered = true;
 
 SELECT * from animals WHERE name not LIKE 'Gabumon';
 
-SELECT * from animals WHERE weight_kg BETWEEN 10.4 and 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg BETWEEN 10.4 and 17.3;
+
+/* Update inside a transaction and rollback*/
+Begin;
+
+UPDATE animals
+set species = 'unespecified';
+
+Rollback;
+
```

Comment:
- [ ] It would be better if we verify that the change was undone

Reply:
- [ ] @NelsonAraujo93 as previously requested kindly add a query to verify the change was made after the rollback.